avocado/core/parameters.py

```diff
@@ -44,7 +44,7 @@ class AvocadoParams(object):
     This is subject of change (separate file, perhaps)
     """
 
-    def __init__(self, leaves, test_id, paths, logger_name=None):
+    def __init__(self, leaves, paths, logger_name=None):
```

Comment:
Yep, I hesitated to remove it to avoid breaking compat with pickle, but with 52 released we can do this sort of changes.

Reply:
Yep.  I also forgot to remove the docstring.  Removing it in v+1.